modules/exploits/multi/http/joomla_http_header_rce.rb

```diff
@@ -5,7 +5,7 @@
 
 require 'msf/core'
 
-class Metasploit3 < Msf::Exploit::Remote
+class Metasploit < Msf::Exploit::Remote
```

Comment:
jeah changing all modules and payloads will result in a lot of conflicts so I changed only one for the initial PR and discussion